src/acl/external/kerberos_ldap_group/support_ldap.cc

```diff
@@ -1115,7 +1115,11 @@ get_memberof(struct main_args *margs, char *user, char *domain, char *group)
                   "%s| %s: DEBUG: Error during initialisation of ldap connection: %s\n",
                   LogTime(), PROGRAM, strerror(errno));
         }
-        bindp = convert_domain_to_bind_path(domain);
+        if (margs->lbind) {
+            bindp = xstrdup(margs->lbind);
+        } else {
+            bindp = convert_domain_to_bind_path(domain);
+        }
```

Comment:
@rousskov thank you, no problem with the changes you made.

Regarding the actual fix, I have decided to duplicate it here purely to minimize the change and allow smaller backports. 

This whole helper dearly needs refactoring to support more targets. May be I'd do that while adding a native FreeIPA LDAP support -- FreeIPA uses RFC2307 while this one is assuming RFC2307bis schema and requires use of a compatibility tree plugin we provide. We want to get rid of that plugin in future so the helper as it is will stop working eventually against FreeIPA.

Reply:
> I have decided to duplicate it here purely to minimize the change and allow smaller backports.
> This whole helper dearly needs refactoring

Understood, thank you. FWIW, while I respectfully disagree with your decision, I do not insist on avoiding code duplication here because I do not have the time to fix this myself and, I assume, this code is full of similar problems (many of which were probably justified by similar excuses!) and needs overall refactoring.